digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/lane/LaneDao.scala

```diff
@@ -0,0 +1,446 @@
+package fi.liikennevirasto.digiroad2.dao.lane
+
+import fi.liikennevirasto.digiroad2.Point
+import fi.liikennevirasto.digiroad2.asset.{BoundingRectangle, LinkGeomSource}
+import fi.liikennevirasto.digiroad2.client.vvh.VVHClient
+import fi.liikennevirasto.digiroad2.lane.{PersistedLane, _}
+import fi.liikennevirasto.digiroad2.oracle.{MassQuery, OracleDatabase}
+import fi.liikennevirasto.digiroad2.service.RoadLinkService
+import slick.driver.JdbcDriver.backend.Database
+import Database.dynamicSession
+import fi.liikennevirasto.digiroad2.dao.Sequences
+import org.joda.time.DateTime
+import slick.jdbc.StaticQuery.interpolation
+import slick.jdbc.{GetResult, PositionedResult}
+import scala.language.implicitConversions
+
+
+case class LaneRow(id: Long, linkId: Long, sideCode: Int, value: LanePropertyRow,
+                   startMeasure: Double, endMeasure: Double, createdBy: Option[String], createdDate: Option[DateTime],
+                   modifiedBy: Option[String], modifiedDate: Option[DateTime], expired: Boolean,
+                   vvhTimeStamp: Long, municipalityCode: Long, laneCode: Int, geomModifiedDate: Option[DateTime])
+
+case class LanePropertyRow(publicId: String, propertyValue: Option[Any])
+
+
+
+class LaneDao(val vvhClient: VVHClient, val roadLinkService: RoadLinkService ){
+
+  val MAIN_LANES = Seq(11,21,31)
```

Comment:
We can have this in every place when implements the case classes relative to lane types like we already talked.
And then can be removed.

Reply:
Done